src/main/java/com.outsystems.plugins.inappbrowser/osinappbrowserlib/views/OSIABWebViewActivity.kt

```diff
@@ -521,31 +581,154 @@ class OSIABWebViewActivity : AppCompatActivity() {
 
         // handle opening the file chooser within the WebView
         override fun onShowFileChooser(
-            webView: WebView?,
-            filePathCallback: ValueCallback<Array<Uri>>?,
-            fileChooserParams: FileChooserParams?
+            webView: WebView,
+            filePathCallback: ValueCallback<Array<Uri>>,
+            fileChooserParams: FileChooserParams
         ): Boolean {
             this@OSIABWebViewActivity.filePathCallback = filePathCallback
-            val intent = fileChooserParams?.createIntent()
+            val acceptTypes = fileChooserParams.acceptTypes.joinToString()
+            val captureEnabled = fileChooserParams.isCaptureEnabled
+            pendingAcceptTypes = acceptTypes
+            pendingCaptureEnabled = captureEnabled
```

Comment:
Minor: The `pending` feels misleading to me, because it's those variables that are used in `retryFileChooser` and `cancelFileChooser`, right?

For me would make more sense for those variables to be `acceptTypes` and `captureEnabled` and then

```suggestion
            acceptTypes = fileChooserParams.acceptTypes.joinToString()
            captureEnabled = fileChooserParams.isCaptureEnabled
```

Reply:
I see, the `pending` was to mean: these will be used when retrying or canceling the file chooser after the permission request.

But they actually aren't needed, we can simplify things and simply always use the same variables.

Refactored it with your suggestion, can you check?